app/views/users/two_factor_authentication_setup/index.html.slim

```diff
@@ -26,6 +25,7 @@ p.mt-tiny.mb0
           = radio_button_tag 'two_factor_setup_form[otp_method]', :voice
           span.indicator
           = t('devise.two_factor_authentication.otp_method.voice')
+  p = t('devise.two_factor_authentication.otp_method.instruction')
```

Comment:
I think since this helper text is associated with the radio buttons, it should remain inside the `fieldset`.  So if it was inside there, after the close of the `label` tag then you could update the `p` to `p.mt1.mb0` and it will not only make more sense semantically, but visually be grouped a little closer to those buttons:

![screen shot 2017-03-30 at 9 51 40 am](https://cloud.githubusercontent.com/assets/1178494/24507600/da2d20b0-152e-11e7-8714-3892e9e3b4fc.png)